Riot/Modules/Spaces/SpaceMenu/SpaceMenuListItemViewData.swift

```diff
@@ -30,19 +38,19 @@ protocol SpaceMenuListItemViewDataDelegate: AnyObject {
 
 /// `SpaceMenuListViewCell` view data
 class SpaceMenuListItemViewData {
-    let actionId: String
+    let actionId: SpaceMenuListItemActionId
     let style: SpaceMenuListItemStyle
     let title: String?
     let icon: UIImage?
-    
+    /// Any value related to the type of data (e.g. `Bool` for `boolean` style, `nil` for `normal` and `destructive` style)
```

Comment:
Ok, so seeing this comment, do you think it would make more sense to drop `value` from here and store this in the `SpaceMenuListItemStyle` case? E.g. something like this:

```swift
enum SpaceMenuListItemStyle {
    case normal
    case toggle(value: Bool)
    case destructive
}
```